ui/src/components/Header.js

```diff
@@ -1,24 +1,32 @@
 import './Header.css';
 
-import React from 'react';
-import AccountCircle from 'material-ui/svg-icons/action/account-circle.js';
+import React, { Component } from 'react';
 import {white} from 'material-ui/styles/colors';
 
-function Header(props) {
```

Comment:
I actually prefer the `function Header(props)` syntax to `class Header extends Component`. When I see the former defined as a function, I immediately understand that I'm looking at a stateless component, since `props` is always immutable. Since this class has no reason to hold `dataset` or `totalCount` as states, what do you think about returning to the original syntax and just calling using `const datasetName = props.datasetName` instead of holding these in states?

Reply:
Done